src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -2,18 +2,27 @@
 @import '../../RangeSlider';
 
 $stacking-order: (
-  output: 10,
+  output: 20,
```

Comment:
I was hesitant about editing these values, especially making `output` and `input` the same. I did this to fix stacking issues such as,

![image](https://user-images.githubusercontent.com/22782157/73298286-07a33080-41db-11ea-923b-033322032dc9.png)

This screenshot is comparing the output bubble to a `primary Connected` element. Doing a quick search for `$stacking-order` shows that most values are exactly, or less than 20.